packages/types/App.d.ts

```diff
@@ -144,6 +144,8 @@ export interface App {
   __template?: string;
   /** Slug of an app needed to be installed before the current app can be added */
   dependencies?: string[];
+  /** Enables video apps to be used for team events */
+  concurrentMeetings?: boolean;
```

Comment:
nit: I am not sure why we are calling it `concurrentMeetings`, wouldn't `teamInstallEnabled` make more sense?

Reply:
Didn't know this. But I still feel that the flag should be named something like `teamInstallEnabled`. You can set teamInstallEnabled based on this conccurentMeetings criteria or any other critera. Hypothetical other criteria might be self hosters wanting to disable certain apps for teams.

It's a nit only.